src/components/dialogmote/FritekstSeksjon.tsx

```diff
@@ -0,0 +1,50 @@
+import { FlexColumn, FlexRow, PaddingSize } from "../Layout";
+import { Field } from "react-final-form";
+import Fritekst from "../Fritekst";
+import { Knapp } from "nav-frontend-knapper";
+import React from "react";
+import {
+  MAX_LENGTH_INNKALLING_FRITEKST,
+  texts,
+} from "./innkalling/DialogmoteInnkallingTekster";
+import styled from "styled-components";
+
+const FritekstWrapper = styled.div`
+  margin-bottom: 4em;
+`;
+
+interface FritekstSeksjonProps {
+  fieldName: string;
+  label: string;
+  handlePreviewClick: () => void;
+}
+
+const FritekstSeksjon = ({
+  fieldName,
+  label,
+  handlePreviewClick,
+}: FritekstSeksjonProps) => (
+  <FritekstWrapper>
+    <FlexRow bottomPadding={PaddingSize.SM}>
+      <FlexColumn flex={1}>
+        <Field<string> name={fieldName}>
+          {({ input }) => (
+            <Fritekst
+              size="medium"
+              maxLength={MAX_LENGTH_INNKALLING_FRITEKST}
+              label={label}
+              {...input}
+            />
+          )}
+        </Field>
+      </FlexColumn>
+    </FlexRow>
+    <FlexRow>
+      <Knapp htmlType="button" onClick={handlePreviewClick}>
+        {texts.forhandsvisning}
```

Comment:
tenker denne teksten kan ligge her i stedet for at den importeres fra DialogmoteInnkallingTekster.